### ISSUE TITLE
Add model description and performance [enhancement]

### ISSUE THREAD

**Issue by g-paras** [keras-team/keras#14683]
Can I add description, accuracy and other information about the trained model so that I can access it later. 
By this I mean I save the model after training and testing using `model.save('model.h5')` , now I want to check the model accuracy on validation set used during training, the only way to do so is to evaluate again using `model.evaluate(X, y)` and I don't want to reapeat the whole process.
It would be nice if somehow we can save the model history (accuracy and loss during training) along with the model, Is that possible ?

**Comment by shubham0204** [comment]
We can serialize the `History` object we get after calling `model.fit()` using `pickle`.

**Comment by g-paras** [comment]
Thank You @shubham0204, it worked 👍🏻
